### DISCUSSION TITLE
rAudio 20230202: Library View - wrong icon placement

### DISCUSSION THREAD

**Discussion by findnix** [rern/rAudio#1223] [Fixed Issues]
Bevore on iPhone 
![image](https://user-images.githubusercontent.com/77154016/216780590-a28e7909-eec0-449a-9f17-a4d765e1c69d.png)
After update to 20230202
![image](https://user-images.githubusercontent.com/77154016/216780640-7f704436-26d6-4c97-b2e5-168214ae3abc.png)

**Comment by klassikmann** [comment]
@findnix 
Looks similar to the now closed [https://github.com/rern/rAudio-1/discussions/1223](url).

**Comment by Vseason** [comment]
This particular issue persists in 20230205.
